addons/stock/models/stock_quant.py

```diff
@@ -141,6 +141,9 @@ def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=Fals
     def write(self, vals):
         """ Override to handle the "inventory mode" and create the inventory move. """
         if self._is_inventory_mode() and 'inventory_quantity' in vals:
+            if any(quant.location_id.usage == 'inventory' for quant in self):
+                # Do nothing when user tries to modify manually a inventory loss
+                return
```

Comment:
i would remove the onchange and raise an user error here

Reply:
note: raising an error blocks the editable list in a wrong state, keep the warning